web/src/pages/GroupDetail.tsx

```diff
@@ -37,6 +41,7 @@ class GroupDetail extends React.Component<GroupProps, Group> {
 
   componentDidMount() {
     this.loadGroup();
+    this.loadDeviceGroup();
```

Comment:
L'appel de la méthode « this.loadDeviceGroup() » n'est pas nécessaire car celle-ci est appelée dans « loadGroup() ».